homeassistant/components/varta_storage/const.py

```diff
@@ -0,0 +1,5 @@
+"""Constants for the VARTA Storage integration."""
+import logging
+
+DOMAIN = "varta_storage"
+_LOGGER = logging.getLogger(__name__)
```

Comment:
This indicates this constant is protected. Shouldn't it just be public?

```suggestion
LOGGER = logging.getLogger(__name__)
```